core/services/vrf/listener_v2.go

```diff
@@ -749,7 +756,9 @@ func (lsn *listenerV2) processRequestsPerSub(
 				"juelsNeeded", p.juelsNeeded.String(),
 				"maxLink", p.maxLink.String(),
 				"gasLimit", p.gasLimit,
-				"attempts", p.req.attempts)
+				"attempts", p.req.attempts,
+				"runningBalance", startBalanceNoReserveLink.String(),
```

Comment:
I've added this field because the `startBalanceNoReservedLink` field doesn't actually update as it's decremented. I also tried overriding the field by using the same name, but it seems like the logger only ever uses the first ever set value for a particular log field.